src/main/java/com/aws/greengrass/logmanager/CloudWatchLogsUploader.java

```diff
@@ -206,7 +209,7 @@
                 .logStreamName(logStreamName)
                 .build();
         try {
-            this.cloudWatchLogsClient.createLogStream(request);
+            this.cloudWatchWrapper.execute(client -> client.createLogStream(request));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

`software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient.createLogStream` API can also throw the following exception types: `InvalidParameterException`, `SdkClientException`, `SdkException`, `AwsServiceException`, `ResourceAlreadyExistsException`, `ServiceUnavailableException`, `ResourceNotFoundException`, `CloudWatchLogsException`, `UnsupportedOperationException`. We recommend handling these uncaught exceptions as well.